src/SigniFlow.Connect/Api/CertifyingCeremonyApi.cs

```diff
@@ -0,0 +1,353 @@
+        /// <param name="certifyingCeremonyRequest"></param>
+        /// <param name="operationIndex">Index associated with the operation.</param>
+        /// <param name="cancellationToken">Cancellation Token to cancel the request.</param>
+        /// <returns>Task of ApiResponse (CertifyingCeremonyResponse)</returns>
+        public async System.Threading.Tasks.Task<SigniFlow.CertifyCeremonyAPI.Client.ApiResponse<CertifyingCeremonyResponse>> CertifyingCeremonyWithHttpInfoAsync(CertifyingCeremonyRequest certifyingCeremonyRequest, int operationIndex = 0, System.Threading.CancellationToken cancellationToken = default(global::System.Threading.CancellationToken))
+        {
+            // verify the required parameter 'certifyingCeremonyRequest' is set
+            if (certifyingCeremonyRequest == null)
+            {
+                throw new SigniFlow.CertifyCeremonyAPI.Client.ApiException(400, "Missing required parameter 'certifyingCeremonyRequest' when calling CertifyingCeremonyApi->CertifyingCeremony");
+            }
+
+
+            SigniFlow.CertifyCeremonyAPI.Client.RequestOptions localVarRequestOptions = new SigniFlow.CertifyCeremonyAPI.Client.RequestOptions();
+
+            string[] _contentTypes = new string[] {
+                "application/json"
+            };
+
+            // to determine the Accept header
+            string[] _accepts = new string[] {
+                "application/json",
+                "application/xml"
+            };
+
+            var localVarContentType = SigniFlow.CertifyCeremonyAPI.Client.ClientUtils.SelectHeaderContentType(_contentTypes);
+            if (localVarContentType != null)
+            {
+                localVarRequestOptions.HeaderParameters.Add("Content-Type", localVarContentType);
+            }
+
+            var localVarAccept = SigniFlow.CertifyCeremonyAPI.Client.ClientUtils.SelectHeaderAccept(_accepts);
+            if (localVarAccept != null)
+            {
+                localVarRequestOptions.HeaderParameters.Add("Accept", localVarAccept);
+            }
+
+            localVarRequestOptions.Data = certifyingCeremonyRequest;
+
+            localVarRequestOptions.Operation = "CertifyingCeremonyApi.CertifyingCeremony";
+            localVarRequestOptions.OperationIndex = operationIndex;
+
+
+            // make the HTTP request
+            var localVarResponse = await this.AsynchronousClient.PostAsync<CertifyingCeremonyResponse>("/CertifyingCeremony", localVarRequestOptions, this.Configuration, cancellationToken).ConfigureAwait(false);
+
+            if (this.ExceptionFactory != null)
+            {
+                Exception _exception = this.ExceptionFactory("CertifyingCeremony", localVarResponse);
+                if (_exception != null)
+                {
+                    throw _exception;
+                }
+            }
+
+            return localVarResponse;
+        }
+
```

Comment:
![RA3 - Spaaaaace](https://github.com/user-attachments/assets/b9fa2edf-91d7-4ac6-a44c-12ac30dafd36)

Reply:
This is in generated code